.github/workflows/build.yml

```diff
@@ -150,6 +150,10 @@ jobs:
             targetPlatform: Android
             vrsdk: Oculus
             extraoptions: -setDefaultPlatformTextureFormat astc -btb-il2cpp -btb-experimental
+          - name: Pico
```

Comment:
Should we call this "Android Pico" to match the other formats?

Reply:
My reasoning was that Pico don't have a desktop headset (yet?) that has a proprietary SDK, so we probably don't need to specify